Off-Top-App/off-top-flutter
# Got a new community license key from syncfusion to keep the focus meter working

**What I did:**

- Replaced the old license key with a new community license key in main.dart.

**How to test:**

- Start the zookeeper server.

- Start the Kafka server.

- In the Spring-Boot dashboard in Visual Studio start off-top.

- Make sure port forwarding is active in chrome, `chrome://inspect/#devices`

- Start the emulator in off-top-flutter.

- In the terminal, enter `flutter run`.

- sign into the Off-Top app.

- The focus meter is displayed, without a pop-up message being displayed about the trial expiration.

- Run server.py in off-top-python.

- Start to record in the Off-Top app.

- The meter should be updating by showing different values and moving the pointer around.
